gui/wxpython/gui_core/forms.py

```diff
@@ -806,11 +806,12 @@ def OnDone(self, event):
             # was closed also when aborted but better is leave it open
             wx.CallLater(2000, self.Close)
 
-    def OnMapCreated(self, name, ltype):
+    def OnMapCreated(self, name, ltype, add=None):
```

Comment:
Since we know for sure the type of the new parameter here, should we add the annotations?
```suggestion
    def OnMapCreated(self, name, ltype, add: bool | None = None):
```

A new line at the top of the file will be asked by ruff so all our supported Python versions accept it.

Reply:
Not sure I understand the comment about new line?

Reply:
It's the new ruff check failure, that needs to add a line before the first import. Try locally, it'll be clear and automatic (either pre commit or locally)

Reply:
I pushed the correction